src/controllers/inventory.js

```diff
@@ -132,4 +138,28 @@ export default function () {
       })
   }
   loadMore()
+  
+  const $exportInventory = $('#export-inventory')
+  const downloadCsv = exportData => {
+    const columns = [`${i18n(i19sku)}`, `${i18n(i19name)}`, `${i18n(i19price)}`, `${i18n(i19unitsInStock)}`]
```

Comment:
```suggestion
    const columns = [i18n(i19sku), i18n(i19name), i18n(i19price), i18n(i19unitsInStock), i18n(i19sales)]
```

Faltou a última coluna e encapsular cada string aí não era necessário (não tem concatenaçaõ nenhuma)